src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (arg) => {
+    if (arg === undefined) {
```

Comment:
I think it is better to check if func parameter is number, cuz in tests we don't have a case where we try to pass any other type than number. So with this our function will be better IMHO.
Ofc you will need to change ur function a bit.